crowbar_framework/spec/spec_helper.rb

```diff
@@ -102,9 +102,8 @@
   config.infer_spec_type_from_file_location!
 
   config.before do
-    ChefObject.stubs(:cloud_domain).returns("crowbar.com")
-
-    Proposal.any_instance.stubs(:properties_template_dir).returns(Rails.root.join("spec/fixtures/data_bags"))
+    allow(ChefObject).to receive(:cloud_domain).and_return("crowbar.com")
+    allow_any_instance_of(Proposal).to receive(:properties_template_dir).and_return(Rails.root.join("spec/fixtures/data_bags"))
```

Comment:
Metrics/LineLength: Line is too long. [127/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)